addons/Media_Recorder/src_webpack/view/SoundIntensity.jsm

```diff
@@ -4,6 +4,19 @@ export class SoundIntensity {
         this.$view = $view;
         this.volumeLevels = 6;
         this.interval = null;
+        this.enableAnalyser = true;
+        this.shouldBeVisible = true;
+    }
+
+    setEnableAnalyser(enableAnalyser) {
+        this.enableAnalyser = enableAnalyser;
+        if (this.shouldBeVisible) {
+            if (this.enableAnalyser) {
+                this.show();
```

Comment:
Skoro sprawdzamy w meteodzie show() czy this.enableAnalyser to nie możnaby tutaj czegoś pominąć i uprościć? Pytanie czy nie dałoby rady sprawdząć tylko czegoś w stylu : 
`this.shouldBeVisible ? this.show() : this.hide()`

Reply:
@maciej-zawlocki-solwit Podbijam to pytanie